---
abstract: 
author_notes:
-
authors:
- Devon A. DeRaad
- admin
- Abdelghafar Alkishe1
- Uzma Ashraf
- Koffi Mensah Ahadji-Dabla
- Claudia Nu√±ez-Penichet
- A. Townsend Peterson
date: ""
doi: "10.1111/mec.16094"
featured: false
image:
projects: []
publication: '*Molecular Ecology 30*: 6468-6485'
publication_short: ""
publication_types:
- "2"
publishDate: "2021-07-26T00:00:00Z"
slides: 
summary: Exploring evolutionary adaptation in a Malaria vector.
tags:
- Evolutionary adaptation
- Disease vector
- Genome-Environment Association
title: 'Genome-environment association methods comparison supports omnigenic adaptation to ecological niche in malaria vector mosquitoes'
url_code: ""
url_dataset: ""
url_pdf: https://onlinelibrary.wiley.com/doi/10.1111/mec.16094
url_poster: ""
url_project: ""
url_slides: ""
url_source: ""
url_video: ""
---

### Abstract

The concept of a fundamental ecological niche is central to questions of geographic distribution, population demography, species conservation, and evolutionary potential. However, robust inference of genomic regions associated with evolutionary adaptation to particular environmental conditions remains difficult due to the myriad of potential confounding processes that can generate heterogeneous patterns of variation across the genome. Here, we interrogate the potential role of genome environment association (GEA) testing as an initial step in building an understanding of the genetic basis of ecological niche. We leverage publicly available genomic data from the *Anopheles gambiae* 1000 Genomes (Ag1000g) Consortium to test the ability of multiple analytically unique GEA methods to handle confounding patterns of genetic variation, control false positive rates, and discern associations with broadly relevant climate variables from random allele frequency patterns throughout the genome. We found evidence supporting the ability of commonly implemented GEA methods to account for confounding patterns of spatial and genetic variation, and control false positive rates. However, we fail to find evidence supporting the ability of GEA tests to reject signals of adaptation to randomly simulated environmental variables, indicating that discerning between true signals of genome environment adaptation and genome environment correlations resulting from alternative evolutionary processes, remains challenging. Because signals of environmental adaptation are so diffuse and confounded throughout the genome, we argue that genomic adaptation to ecological niche is likely best understood under an omnigenic model wherein highly interconnected, genome-wide gene regulatory networks shape genomic adaptation to key environmental conditions.
{style="text-align: justify;"}

<br>
